docs/marionette.region.md

```diff
@@ -233,6 +235,8 @@ myView.addRegions({
 });
 ```
 
+[Live example](http://jsfiddle.net/charlyjazz/gze5ykov/49/)
```

Comment:
Does this need to be switched out for a `marionettejs` account URL?

Reply:
I've forked at http://jsfiddle.net/marionettejs/kjvzdyd6/ to use marionettejs account